config.go

```diff
@@ -1 +1,105 @@
 package main
+
+import (
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil"
+	flags "github.com/btcsuite/go-flags"
+	"path/filepath"
+	"io/ioutil"
+	"os"
+)
+
+const (
+	defaultConfigFilename = "lnwallet.conf"
+	defaultDataDirname    = "test_wal"
+	defaultRPCPort        = 10009
+	defaultSPVMode        = false
+	defaultPeerPort       = 10011
+	defaultBTCDHost       = "localhost:18334"
+	defaultBTCDUser       = "user"
+	defaultBTCDPass       = "passwd"
+	defaultBTCDCACertPath = ""
+	defaultUseRegtest     = false
+	defaultSPVHostAdr     = "localhost:18333"
+	defaultBTCDNoTLS      = false
+)
+
+var (
+	lnwalletHomeDir   = btcutil.AppDataDir("lnwallet", false)
+	defaultConfigFile = filepath.Join(lnwalletHomeDir, defaultConfigFilename)
+	defaultDataDir    = filepath.Join(lnwalletHomeDir, defaultDataDirname)
+)
+
+type config struct {
```

Comment:
After some thought, I think configuration management will be more stream lined if lnwallet's config file isn't embedded after all, so you can safely ignore the above comment :)